crates/teloxide-core/src/types/reply_parameters.rs

```diff
@@ -0,0 +1,50 @@
+use serde::{Deserialize, Serialize};
+
+use crate::types::{MessageId, Recipient};
+
+/// Describes reply parameters for the message that is being sent.
+#[serde_with::skip_serializing_none]
+#[derive(Default, Clone, Debug, Eq, Hash, PartialEq, Serialize, Deserialize)]
+pub struct ReplyParameters {
+    /// Identifier of the message that will be replied to in the current chat,
+    /// or in the chat _chat\_id_ if it is specified
+    pub message_id: MessageId,
```

Comment:
I was testing my TBA 7.1 PR and noticed a bug:
`ERROR teloxide::error_handlers          > An error has occurred in the dispatcher: Api(Unknown("Bad Request: field \"message_id\" must be a Number"))`

Shouldn't be `#[serde(flatten)]` on `message_id` here?

Reply:
Yep, you're right

Reply:
I fixed it in all places